utils/iter-support.js

```diff
@@ -0,0 +1,10 @@
+function iterSupport(compat, browser) {
+  if (browser in compat.support) {
+    const data = compat.support[browser];
+    return Array.isArray(data) ? data : [data];
+  }
+
+  return [];
```

Comment:
An entry with `version_added` null would make this even more like the real data.

Reply:
Ooh, that's a great idea. Done in 59c11d03.